frontend/components/Documents.js

```diff
@@ -90,28 +90,42 @@ const Documents = () => {
 
     const docInfo = (doc, i) => {
         return (
-            <ul> Document {i}
-                {Object.keys(doc).map((attribute, i) => (
-                    <li key={i}>{attribute}: {doc[attribute]}</li>
-                ))}
-            </ul>
+            <div className="card" key={i}>
```

Comment:
I think we can remove the `key` attribute here. Those are only necessary when we're creating multiple elements via iteration or mapping. In this case, this `docInfo` function is simply a convenience wrapper for displaying a few pieces of content. The surrounding `<div>` already has its `key` attribute appropriately.